src/api/rooms/getRoomPlaying.ts

```
@@ -65,8 +65,14 @@ export const getRoomPlaying = async (roomId: number): Promise<RoomPlayingRespons
   try {
     const response = await apiClient.get<RoomPlayingResponse>(`/rooms/${roomId}/playing`);
     return response.data;
-  } catch (error) {
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)